.install/python-macro.yaml

```diff
@@ -0,0 +1,163 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  This CFN Template allows you to install the AWS CFN Macro as a Lambda Function to execute ECS Compose-X directly
+  from within your AWS CloudFormation templates.
+
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Metadata:
+  Maintainer: john@compose-x.io
+  Version: 2021-01-25
+  AWS::CloudFormation::Interface:
+    ParmeterLabels:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Metadata Interface has invalid property ParmeterLabels